acquire/acquire.py

```diff
@@ -557,6 +557,34 @@ def _run(cls, target: Target, cli_args: argparse.Namespace, collector: Collector
             mem_dump_errors_path.unlink()
 
 
+@register_module("--winmem-files")
+class WinMemFiles(Module):
+    DESC = "Windows memory files"
+    SPEC = [
+        ("file", "sysvol/pagefile.sys"),
+        ("file", "sysvol/hiberfil.sys"),
+        ("file", "sysvol/swapfile.sys"),
+        ("file", "sysvol/windows/memory.dmp"),
+        ("dir", "sysvol/windows/minidump"),
+    ]
+
+    @classmethod
+    def get_spec_additions(cls, target: Target, cli_args: argparse.Namespace) -> Iterator[tuple]:
+        spec = set()
+
+        page_key = "HKLM\\SYSTEM\\CurrentControlSet\\Control\\Session Manager\\Memory Management"
+        for reg_key in target.registry.iterkeys(page_key):
+            for page_path in reg_key.value("ExistingPageFiles").value:
+                spec.add(("file", target.resolve(page_path)))
```

Comment:
Since Dissect 3.8 it is now possible to use globbing for the registry, so try changing the code to the following in order to eliminate the first for-loop:

```suggestion
        page_key = "HKLM\\SYSTEM\\CurrentControlSet\\Control\\Session Manager\\Memory Management\\*"
        for page_path in page_key.value("ExistingPageFiles").value:
             spec.add(("file", target.resolve(page_path)))
```